library/src/main/java/com/fstyle/library/GravityEnum.java

```diff
@@ -0,0 +1,42 @@
+package com.fstyle.library;
+
+import android.annotation.SuppressLint;
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.view.Gravity;
+import android.view.View;
+
+public enum GravityEnum {
+  START,
+  CENTER,
+  END;
+
+  private static final boolean HAS_RTL =
```

Comment:
ERROR: 'member def modifier' have incorrect indentation level 2, expected level should be 4.